src/Sylius/Bundle/ApiBundle/Doctrine/QueryItemExtension/OrderMethodsItemExtension.php

```diff
@@ -49,7 +50,17 @@ public function applyToItem(
             return;
         }
 
-        if ($operationName !== Request::METHOD_DELETE) {
+        if (
+            $operationName === Request::METHOD_GET ||
+            $operationName === ApiShopRequestTypes::SHOP_GET ||
+            $operationName === ApiShopRequestTypes::ADJUSTMENTS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::ITEMS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::ITEMS_ADJUSTMENTS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::SHIPMENTS_METHODS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::PAYMENTS_METHODS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::SHIPMENTS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::PAYMENTS_GET_SUBRESOURCE
+        ) {
             return;
         }
 
```

Comment:
Below this line there is `applyToItemForDeleteMethod` method which should be renamed to `applyUserRulesToItem`